RELEASE.md

```diff
@@ -1,11 +1,43 @@
-# Release 1.5.0
+# Release 1.6.0
 
 ## Breaking Changes
 * Prebuilt binaries are now built against CUDA 9.0 and cuDNN 7.
```

Comment:
This has not changed since 1.5. So technically it is not a breaking change. But I like it to be easy to find.
So While I have some objections to the section, I like it being in the release notes.